core/packages/test/config/launch-config.toml

```
@@ -0,0 +1,95 @@
+[settings]
+node_spawn_timeout = 120
+provider = "native"
+timeout = 600
+
+## Rococo
+[relaychain]
+chain = "rococo-local"
+default_command = "{{output_bin_dir}}/polkadot"
+default_args = ["-lparachain=debug","--beefy","--enable-offchain-indexing=true","--pruning","archive"]
+
+[relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
+max_validators_per_core = 2
+needed_approvals = 1
+validation_upgrade_cooldown = 2
+validation_upgrade_delay = 2
+
+[[relaychain.nodes]]
+name = "rococo-alice"
+rpc_port = 30444
+validator = true
+ws_port = 9944
+
+[[relaychain.nodes]]
+name = "rococo-bob"
+validator = true
+
+[[relaychain.nodes]]
+name = "rococo-charlie"
+validator = true
+
+[[relaychain.nodes]]
+name = "rococo-dave"
+validator = true
```

Should we include the RPC and websocket ports here, like with the rococo-alice node?
```suggestion
[[relaychain.nodes]]
name = "rococo-bob"
rpc_port = 30555
validator = true
ws_port = 9955

[[relaychain.nodes]]
name = "rococo-charlie"
rpc_port = 30666
validator = true
ws_port = 9966

[[relaychain.nodes]]
name = "rococo-dave"
rpc_port = 30777
validator = true
ws_port = 9977
```

---

https://github.com/Snowfork/snowbridge/pull/754#issuecomment-1384709684